src/io/flutter/run/daemon/DaemonApi.java

```diff
@@ -80,6 +81,10 @@
     return send("app.stop", new AppStop(appId));
   }
 
+  CompletableFuture<Pair<String, Integer>> devToolsServe() {
```

Comment:
nit: create a little Value class for this instead of using Pair.
With Pair it isn't obvious what the two fields are but it would be really obvious with a simple data class with a `host` and `port` field.